### ISSUE TITLE
Localization support is missing.

### ISSUE THREAD

**Issue by Lysarion** [TurkeyTickle/mhw-builder#40]
Greetings.

Title say it all: your web-app does not include localization support. Which is kind of annoying since MHW teams have been a bit lazy and some translations are horrible and make this tool hard to use for whoever is not running the game in english (especially infuriating for mistranslated name of gear or jewels).

Seeing as how you implemented your data storage, i think the simplest way to implement localization would be slightly changing the folders tree to add language-specific subdirectories in "mhw-builder/src/assets" (en/fr/ger/jp, etc) and adding a small "translate" function that just switch the folder it reads data in (and which default to the "en" folder whenever it does not find the appropriate translation).

(I'm french, and should you add localization support, i would be glad to help translate the french one.)

PS: Also, couldn't add the "enhancement" label.

**Comment by TurkeyTickle** [comment]
It sounds like that would be the easiest way to add localization, but if I'm understanding your suggestion correctly, that would also mean that there would be copies of all the data in each language folder which I think would become difficult to maintain and keep in sync between languages pretty quickly. 

I think it's probably worth putting a little extra work to share most of the data (attack values, resistance values, etc.) across all languages, but allow names and descriptions to be overridden for each language.

Either way, it should be fairly simple to add, but I probably won't be able to get to it until this weekend. Also, it would great to have you help out on the french translation!

**Comment by TurkeyTickle** [comment]
I've added localization support in the develop branch. It's not perfect yet, but should be good enough to get started. 

When the page is loaded, your browser's primary locale is stored in local storage and determines which resources are loaded - I'll add some way for users to pick their locale manually later. I also added a "src/assets/fr-FR" folder with files that contain ids and translatable data. All names, descriptions, and tags (tags are not actually displayed anywhere in the UI, but are used for item search) in this folder can be translated, but all ids should be left alone because they're used to match back to the "master" data in the root assets folder.

There's still more work to do so I'll leave this issue open for now, but I think it's ready for you to start translating, so let me know if you have any questions.
